index.js

```diff
@@ -653,32 +656,38 @@ class ClientServiceBase {
   }
 
   /**
-   * @param {ContractExecutionResult} result1
-   * @param {ContractExecutionResult} result2
-   * @return {Boolean}
-   * @throws {ClientError}
+   * @param {proto.rpc.ContractExecutionResponse} response1
+   * @param {proto.rpc.ContractExecutionResponse} response2
+   * @return {boolean}
    */
-  _validateResponses(result1, result2) {
-    // We assume that JSON.parse() creates the identically-ordered object
-    // in fromGrpcContractExecutionResponse() if the execution result is same.
-    if (JSON.stringify(result1.getResult()) !==
-        JSON.stringify(result2.getResult()) ||
-        result1.getProofs().length !== result2.getProofs().length) {
+  _validateResponses(response1, response2) {
+    const proofs1 = response1
+        .getProofsList()
+        .map((p) => AssetProof.fromGrpcAssetProof(p));
+    const proofs2 = response2
+        .getProofsList()
+        .map((p) => AssetProof.fromGrpcAssetProof(p));
+
+    if (
+      response1.getContractresult() !== response2.getContractresult() ||
```

Comment:
Sorry for the late feedback, but why is this method not properly camel-cased?
Should it be `getContractResult()`?

Reply:
@feeblefakie 
I revised this part according to the new scalar.proto in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/79/commits/0a36461abd71d4caa4b22169811e4aaf59d22e5d

Please take a look!
Thank you.